src/main/java/emissary/core/EmissaryRuntimeException.java

```diff
@@ -0,0 +1,38 @@
+package emissary.core;
+
+/**
+ * This is the top of the exception hierarchy for Emissary All emissary exceptions extend from here
```

Comment:
Not sure what description we want here, but this one's not accurate.  With the addition of this class, I'm not sure it's still true in the  `EmissaryException` class either.